assets/js/components/ActivateAnalyticsCTA.js

```diff
@@ -47,9 +47,13 @@ export default function ActivateAnalyticsCTA( { children } ) {
 		select( CORE_MODULES ).isModuleActive( 'analytics-4' )
 	);
 
-	const analyticsModuleAvailable = useSelect( ( select ) =>
-		select( CORE_MODULES ).isModuleAvailable( 'analytics-4' )
-	);
+	const analyticsModuleAvailable = useSelect( ( select ) => {
+		const { isModuleAvailable } = select( CORE_MODULES );
+		return (
+			isModuleAvailable( 'analytics-4' ) &&
+			!! select( MODULES_ANALYTICS_4 )
```

Comment:
This added check on the store being available is necessary to ensure that the store is available since we're not using the usual logic to `getModuleStoreName` instead. This would be a bit of a heavier change here so this is ok for now as we have other similar components to update. Ideally we wouldn't need to check this at all, and this component could also live in the module, and only run if it was available.